CONTRIBUTORS.md

```diff
@@ -0,0 +1,89 @@
+## Contributing
+
+### Setup
```

Comment:
The heading levels should be moved up now that this section is a standalone document (`##` :arrow_right: `#`, `###` :arrow_right: `##`, `####` :arrow_right: `###`).

Reply:
done